packages/shared/providers.yaml

```diff
@@ -73,11 +73,14 @@ adobe-umapi:
             after: 'retry-after'
         base_url: https://usermanagement.adobe.io
     docs: https://docs.nango.dev/integrations/all/adobe
+    post_connection_script: adobeUmapiPostConnection
+    docs_connect: https://docs.nango.dev/integrations/all/adobe-umapi/connect
     connection_config:
         clientId:
             type: string
-            title: Client ID
-            description: The Client ID from the OAuth Server-to-Server enabled project on developer portal
+            title: ''
+            description: ''
```

Comment:
why emptying the title and description?

Reply:
They are both needed in order for validation to pass, but at the same time we are not going to display it to the user hence no need of having any text as it is automated with the `postConnection` script.

Reply:
yeah indeed. At some point we might still want to document those values in the dashboard so we'll see then